### PR TITLE
add cargo.toml lint section way of adding lints in the book

### PR DESCRIPTION
**Pull request by lucarlig**
changelog: add cargo.toml lint section way of adding lints in the book

as from [discussion on zulip](https://rust-lang.zulipchat.com/#narrow/stream/257328-clippy/topic/.E2.9C.94.20clippy.20config)

### CODE REVIEW

File: book/src/configuration.md [hunk 1]
```diff
@@ -79,6 +79,24 @@ interested in:
 cargo clippy -- -A clippy::all -W clippy::useless_format -W clippy::...
 ```
 
+The last way to allow/disallow lints is to use `Cargo.toml` using [The lints section](`Cargo.toml` using [The lints section](`Cargo.toml` using [The lints section](https://github.com/rust-lang/rust-clippy/blob/master/book/src/configuration.md#allowingdenying-lints)):
```
Comment (flip1995) [inline book/src/configuration.md:82]: TBH all this looks like a LLM fuck-up :sweat_smile: 

Please just link to the section in the cargo book and add at most one concrete (!) example.

**Review by flip1995 — CHANGES_REQUESTED** [review]

File: book/src/configuration.md (hunk 1 above)
Comment (lucarlig) [reply book/src/configuration.md:82]: no all me, chatgpt explanation was way too long, couldn't use it

**Comment by flip1995** [comment]
Please squash your commits into one after applying the above suggestions. Then this is good to go :+1:

**Review by flip1995 — APPROVED** [review]
Thanks! I took this opportunity to clean up this whole section and make the wording more consistent overall.

**Pull request merged**